executor/cvsTask/cvstask.go

```diff
@@ -128,28 +128,28 @@ func (task *cvsTask) CloseImpl(ctx context.Context) error {
 func (task *cvsTask) Receive(ctx context.Context) error {
 	conn, err := grpc.Dial(task.srcHost, grpc.WithInsecure())
 	if err != nil {
-		log.L().Info("cann't connect with the source address ", zap.Any("message", task.srcHost))
+		log.L().Error("cann't connect with the source address ", zap.Any("message", task.srcHost))
 		return err
 	}
 	client := pb.NewDataRWServiceClient(conn)
 	defer conn.Close()
 	reader, err := client.ReadLines(ctx, &pb.ReadLinesRequest{FileName: task.srcDir, LineNo: task.index})
 	if err != nil {
-		log.L().Info("read data from file failed ", zap.Any("message", task.srcDir))
+		log.L().Error("read data from file failed ", zap.Any("message", task.srcDir))
 		return err
 	}
 	for {
 		reply, err := reader.Recv()
 		if err != nil {
-			log.L().Info("read data failed", zap.Any("error:", err.Error()))
+			log.L().Error("read data failed", zap.Error(err))
 			if !task.isEOF {
 				task.cancelFn()
 			}
 			return err
 		}
 		if reply.IsEof {
 			log.L().Info("Reach the end of the file ", zap.Any("fileName:", task.srcDir))
-			task.isEOF = true
+			close(task.buffer)
```

Comment:
if the server crash  in the syncing process, the status should be set error

Reply:
I think the status assignment is here https://github.com/hanfei1991/microcosm/pull/151/files#diff-d9aaad6e272ef0ddb3d4b3e96e3d52db823421edb128f729d16bac74fcbc1253R87